CHANGELOG.md

```diff
@@ -4,5 +4,7 @@
 
 * Allow for boolean false as default setting value ([yuszuv](https://github.com/yuszuv))
 * Convert nested configs to nested hashes with `Config#to_h` ([saverio-kantox](https://github.com/saverio-kantox))
+* Introduce `Configurable.finalize!` which freezes config and its dependencies ([qcam](https://github.com/qcam))
```

Comment:
Can you add this under a `## Added` title please

P.S. Sorry to be a pain and keep asking for changes